app/views/stations/show.html.erb

```diff
@@ -1,16 +1,18 @@
 <main>
-  <div class="station_show">
-    <h1><%= @station.name %></h1>
-    <h3>Dock Count: </h3><p><%= @station.dock_count %></p>
-    <h3>City: </h3><p><%= @station.city %></p>
-    <h3>Installation Date: </h3><p><%= @station.built %></p>
-    <h3>Latitude: </h3><p><%= @station.lat %>°</p>
-    <h3>Longitude: </h3><p><%= @station.long %>°</p>
-    <% if current_user && current_user.admin? %>
-        <div class="table-buttons float-right">
-          <%= link_to '<i class="far fa-edit"></i>'.html_safe, edit_admin_station_path(@station) %>
-          <%= link_to '<i class="far fa-trash-alt"></i>'.html_safe, admin_station_path(@station), method: :delete, data: { confirm: "Are you sure? You will permanently delete this station."} %>
-        </div>
-    <% end %>
+  <div class="card-container">
+    <div class="card-info">
+      <h1><%= @station.name %></h1>
+      <h3>Dock Count: </h3><p><%= @station.dock_count %></p>
+      <h3>City: </h3><p><%= @station.city %></p>
+      <h3>Installation Date: </h3><p><%= @station.built %></p>
+      <h3>Latitude: </h3><p><%= @station.lat %>°</p>
+      <h3>Longitude: </h3><p><%= @station.long %>°</p>
+      <% if current_user && current_user.admin? %>
```

Comment:
At some point we should refactor this and place it in the ApplicationController, eg 
```rb
def current_admin?
  current_user && current_user.admin?
end
```